experimental/framework/frs-client/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "@fluidframework/build-common/ts-common-config.json",
+    "exclude": [
+        "dist",
+        "node_modules"
+    ],
+    "compilerOptions": {
+        "module": "esnext",
```

Comment:
FYI this will cause `npm run build:commonjs` to emit esnext style bundles, I have just opened a PR address it in related experimental packages https://github.com/microsoft/FluidFramework/pull/6206